cmd/microcloud/cluster_members.go

```diff
@@ -133,6 +145,23 @@ func (c *cmdClusterMembersList) Run(cmd *cobra.Command, args []string) error {
 	return cli.RenderTable(c.flagFormat, header, data, clusterMembers)
 }
 
+func (c *cmdClusterMembersList) listLocalClusterMembers(m *microcluster.MicroCluster) error {
+	members, err := m.GetDqliteClusterMembers()
+	if err != nil {
+		return err
+	}
+
+	data := make([][]string, len(members))
+	for i, member := range members {
+		data[i] = []string{strconv.FormatUint(member.DqliteID, 10), member.Name, member.Address, member.Role}
+	}
+
+	header := []string{"ID", "NAME", "ADDRESS", "ROLE"}
```

Comment:
Maybe change the `ID` to `DQLITE ID` to make it clear what this is referring to?